### ISSUE TITLE
[Feature Request] Option to filter out entities localization and only localize narration elements and menus

### ISSUE THREAD

**Issue by M0Rf30** [diasurgical/devilutionX#6254]
A guy from devilutionX italian discord contacted me to know if it's possible to localize only the speeches, without involving itemDescriptions or itemNames or monsterNames, only dialogues between characters and things strictly related to the narration and menu system.
do you think it is feasible?

WHY
Motivation is closely related to deep habits around the names of entities, reported here and there on wikis and communities.
This impacts also on colloquial references between gamers.

PROPOSAL
have a filter option to enable/disable entities localization

TLDR
filter out entities localizations and only localize narration elements and menus

https://github.com/diasurgical/devilutionX/pull/6052#issuecomment-1602944627

**Comment by AJenbo** [comment]
@M0Rf30 would it be ok to still translate the item stats, just not the name?

**Comment by M0Rf30** [comment]
> @M0Rf30 would it be ok to still translate the item stats, just not the name?

Yeah I think would be ok.

**Comment by julealgon** [comment]
> WHY
Motivation is closely related to deep habits around the names of entities, reported here and there on wikis and communities.
This impacts also on colloquial references between gamers.

How are other games translated to Italian though?

I think I get where you guys are coming from as I also find some of the pt-BR names for items extremely weird and out of place, but at the same time... other games translated to pt-BR also follow that and translate everything. And the reason they seem to out of place to us is precisely because games were almost _never_ translated to pt-BR before a few years ago, so it's natural that things we never have contact with (i.e. medieval weapons/armor) sound strange.

Without precedent, I find it hard to justify doing this "customization" to the translation... it's just going to make it more complicated for a niche use case.

**Comment by AJenbo** [comment]
I think it's worth considering that most text, even in Italien (or other languages, this is not the only locale group that has requested such an option) is written using the English names as that is how they appeared to players.

**Comment by M0Rf30** [comment]
> > WHY
> > Motivation is closely related to deep habits around the names of entities, reported here and there on wikis and communities.
> > This impacts also on colloquial references between gamers.
> 
> How are other games translated to Italian though?
> 
> I think I get where you guys are coming from as I also find some of the pt-BR names for items extremely weird and out of place, but at the same time... other games translated to pt-BR also follow that and translate everything. And the reason they seem to out of place to us is precisely because games were almost _never_ translated to pt-BR before a few years ago, so it's natural that things we never have contact with (i.e. medieval weapons/armor) sound strange.
> 
> Without precedent, I find it hard to justify doing this "customization" to the translation... it's just going to make it more complicated for a niche use case.

I don't think it's related to how the games are translated into Italian or how strange medieval weapon names sound.
We're talking about a game that has only ever existed in English, it has an established audience for decades.
If it's about putting an option to switch between the two modes (with default on 'disabled'), why not offer it ?Most of the strings are already categorized.

**Comment by julealgon** [comment]
> If it's about putting an option to switch between the two modes (with default on 'disabled'), **why not offer it** ?Most of the strings are already categorized/categorizable.

Increased complexity both in code and UI to handle the extra option would be my initial answer. But also, a "partial translation" looks unprofessional and jarring, so I don't support it on that principle as well.

I personally think the argument that "but this is how people always saw the names" is weak here: they also always saw the story text that way. At that point, I'd honestly just suggest doing a "story run" of the game using your localization of choice, and then switching back to English for subsequent playthroughs.
